stacks-signer/CHANGELOG.md

```diff
@@ -11,10 +11,25 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
+## [3.0.0.0.2.0]
```

Comment:
Typo here, should have .3

Reply:
ec48c5833a49d65540e8d9c943db260f3f9d3f19